tools/clang/include/clang/Basic/DiagnosticGroups.td

```diff
@@ -781,4 +781,9 @@ def FutureCompat : DiagGroup<"future-compat">;
 // HLSL Change Starts - HLSL diagnostic groups
 def HLSLEffectsSyntax : DiagGroup<"effects-syntax">;
 def HLSLForRedefinition : DiagGroup<"for-redefinition">;
+def HLSLUnsupportedStatement : DiagGroup<"unsupported-statement">;
```

Comment:
I think this should probably be:
```
def HLSLAttributeUnsupportedStatement : DiagGroup<"attribute-unsupported-statement">;
```
Though that's long.  Feel free to shorten `attribute` to `attr`, but then the next one `attribute-type` should probably be made consistent with that.

Reply:
Or better:
```
def HLSLAttributeStatement : DiagGroup<"attribute-statement">;
```

Reply:
Fixed